src/framework/components/camera/component.js

```diff
@@ -289,6 +289,10 @@ class CameraComponent extends Component {
         }
     }
 
+    set renderSceneColorMap(value) {
```

Comment:
Just a suggestion....but maybe `renderColorMap` and `renderDepthMap` are a bit more succinct? The `Scene` part seems kinda superfluous.

Reply:
Sure, we can rename it but then we should rename the getter too, which would require an API change :)

Reply:
I think I added 'scene' to the name on purpose .. otherwise it's not very clear what color map the camera should render. I sounded almost like a property to enable / disable color and depth write on the camera.

Reply:
We could also call it something completely different, like depthGrabPass/colorGrabPass.